tests/MongoDB.Driver.Tests/Specifications/Runner/MongoClientJsonDrivenTestRunnerBase.cs

```diff
@@ -309,7 +309,7 @@ protected void SetupAndRunTest(JsonDrivenTestCase testCase)
 
         protected void VerifyCollectionData(IEnumerable<BsonDocument> expectedDocuments)
         {
-            var database = DriverTestConfiguration.Client.GetDatabase(DatabaseName);
+            var database = DriverTestConfiguration.Client.GetDatabase(DatabaseName).WithReadConcern(ReadConcern.Local);
```

Comment:
Change made so that the convenient API for transaction tests also benefit from this change

Reply:
Acknowledged